mst/migrate-to-mst.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Migrating an existing TimescaleDB database to Managed Service for TimescaleDB
+excerpt: Migrating an existing TimescaleDB database to Managed Service for TimescaleDB
+product: mst
+keywords: [migrate, databse]
+tags: [ingest, backup, restore]
+---
+
+# Migrate the entire database to Managed Service for TimescaleDB
+
+You can migrate your data from self-hosted TimescaleDB to Managed Service for
+TimescaleDB and use exclusive features, including separate scaling for compute
+and storage requirements, first-class multi-node support, and more.
+
+Managed Service for TimescaleDB creates a database named `defaultdb` and a
+default user account named `tsdbadmin`. You can use the Web console to create
+additional users and databases using the `Users` and `Databases` tabs. 
+
+You can switch between different plan sizes in Managed Service for TimescaleDB
+but for the duration of the dumping process choose a plan size that is large
+enough for the task. This allows you to limit the downtime during the migration
```

Comment:
Do we have a suggestion for what plan size is "large enough"?

Reply:
> Do we have a suggestion for what plan size is "large enough"?

good question. Let me find out.

Reply:
@alexthinath  could you please let know what plan size is "large enough"?

Reply:
Perhaps we update the text to say something like "the same storage size or slightly larger than the currently allocated plan"